routers/api/v1/repo/transfer.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
+// Copyright 2018 The Gitea Authors. All rights reserved.
```

Comment:
2020?

Reply:
Only this one I think.

Reply:
it was originaly includet in repo.go there it comes form ... will change